pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -200,6 +200,70 @@ def test_td_rsub_numeric_raises(self):
         with pytest.raises(TypeError):
             2.0 - td
 
+    def test_td_sub_timedeltalike_object_dtype_array(self):
+        # GH 21980
+        arr = np.array([Timestamp('20130101 9:01'),
+                        Timestamp('20121230 9:02')])
+        exp = np.array([Timestamp('20121231 9:01'),
+                        Timestamp('20121229 9:02')])
+        res = arr - pd.Timedelta('1D')
+        tm.assert_numpy_array_equal(res, exp)
+
+    def test_td_sub_mixed_timedeltalike_object_dtype_array(self):
+        # GH 21980
+        now = pd.Timestamp.now()
+        arr = np.array([now,
+                        pd.Timedelta('1D')])
+        exp = np.array([now - pd.Timedelta('1D'),
+                        pd.Timedelta('0D')])
+        res = arr - pd.Timedelta('1D')
+        tm.assert_numpy_array_equal(res, exp)
+
+    def test_td_sub_mixed_most_timedeltalike_object_dtype_array(self):
```

Comment:
It seems like this makes `test_td_sub_mixed_timedeltalike_object_dtype_array` above unnecessary.  Am I missing something?

Reply:
no you're right, on further inspection i removed it